internal/checker/terraform/rules/azure/azurerm_virtual_machine_agent_installed.rego

```diff
@@ -0,0 +1,57 @@
+# METADATA
+# title: "Verify VM Agent Installation"
+# description: "Ensure that the VM Agent is correctly installed on virtual machines for enhanced management and monitoring"
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/linux_virtual_machine
+# custom:
+#   id: CB_TFAZR_153
+#   severity: LOW
+package lib.terraform.CB_TFAZR_153
+
+import future.keywords.in
+
+supportedResources := [
+	"azurerm_windows_virtual_machine",
+	"azurerm_windows_virtual_machine_scale_set",
+	"azurerm_linux_virtual_machine_scale_set",
+	"azurerm_linux_virtual_machine",
+]
+
+isvalid(block) {
+	block.Type == "resource"
+	some label in block.Labels
+	label in supportedResources
+}
+
+resource[resource] {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource[resource] {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
+pass[resource] {
+	some resource in input
+	isvalid(resource)
+	not resource.Attributes.provision_vm_agent
```

Comment:
this should be
resource.Attributes.provision_vm_agent == true

Reply:
Thank you for noticing, pr updated.